gateway/slack.go

```diff
@@ -174,7 +181,7 @@ func (sc *SlackClient) Poop(chans *ClientChans) {
 				messageData, ok := event.Data.(*slack.MessageEvent)
 				if !ok {
 					chans.IncomingChan <- newSlackMessageEvent(
```

Comment:
Does it make sense to have a special cased `newTanyaSlackMessageEvent` (or something) with the tanya fields pre-filled

Reply:
probably, yeah.